projects/pipelines/roles/pipelines_run_kfp_notebook/defaults/main/config.yml

```diff
@@ -38,3 +38,6 @@ pipelines_run_kfp_notebook_capture_artifacts: true
 
 # If True, captures the Prometheus DB of the systems.
 pipelines_run_kfp_notebook_capture_prom_db: false
+
+# Does not change the test, but needed to decide which artifacts to save
+pipelines_run_kfp_notebook_user_count: 0
```

Comment:
@drewrip , can you change that to something like `capture_extra_artifacts`, so that the role doesn't have an "indirect" parameter?

(maybe you can change it as well in other the role where you have `is_small_test`, IIRC)